qtc_packaging/debian_fremantle/changelog

```diff
@@ -1,3 +1,8 @@
+kodimote (2.0.2-fremantle0) unstable; urgency=low
+  * transition from xbmcremote to kodimote
+
+ -- Michael Zanetti <michael_zanetti@gmx.net>  Sun, 01 Mar 2015 20:00:00 +0200
```

Comment:
Wrong name? :smiley: 

And the change log would be (a lot) bigger. Even if the last generic release was a looong time ago and it hasn't been maintained since, there have been a lot of changes in the lib which also affect the app. Like fixing the random crasher (mostly when opening now playing, dis/reconnecting, ...) and additions to the different libraries (don't know if it already contains the "Live TV" entry in the main menu? which has also changed a lot (browsing channels, viewing EPG as "details", browsing recordings, ...))